backend/capellacollab/projects/toolmodels/routes.py

```diff
@@ -144,7 +144,8 @@ def patch_tool_model(
             raise fastapi.HTTPException(
                 status_code=status.HTTP_409_CONFLICT,
                 detail={
-                    "reason": "A model with a similar name already exists."
+                    "reason": "A model with a similar name already exists.",
```

Comment:
I'd consider being more explicit e.g. "A model with the name '{new_model.name}' already exists."

Reply:
There might be some slug collisions even if the name is different. Also in this case, we'd enter this error case.
However, as the user doesn't really care about it, I used your proposal and clarified it in the technical section.